.packit.yaml

```diff
@@ -0,0 +1,44 @@
+actions:
+  create-archive:
+    - './autogen.sh --disable-modules --disable-daemon'
+    # FIXME: just calling `make dist` doesn't build udisks/libudisks2.la for gtk-doc
+    - 'make'
+    - 'make dist'
+    - 'bash -c "ls *.tar*"'
```

Comment:
So first I thought making a tarball from git sources directly and calling `autogen.sh` within the spec file might be a cleaner way, however then I realized the spec file wouldn't work against proper dist tarball as there's no `autogen.sh` then. Sure, a bunch of if-else stuff might do the trick but it's not a clean way either.

Reply:
Yeah, I thought the same -- it's quite nice to be able to build a package straight from a `git archive` tarball, without autoconfiscation. But indeed that then wouldn't have ./configure, Makefile.in, etc.